README.md

```diff
@@ -0,0 +1,55 @@
+<div align="center">
+
+# slurm charms
+
+Manage HPC clusters using [Juju](https://juju.is) and the [SLURM](https://slurm.schedmd.com/overview.html) workload manager.
+
+[![CI](https://github.com/charmed-hpc/slurm-charms/actions/workflows/ci.yaml/badge.svg)](https://github.com/charmed-hpc/slurm-charms/actions/workflows/ci.yaml/badge.svg)
+[![Release](https://github.com/charmed-hpc/slurm-charms/actions/workflows/release.yaml/badge.svg)](https://github.com/charmed-hpc/slurm-charms/actions/workflows/release.yaml/badge.svg)
+[![Matrix](https://img.shields.io/matrix/ubuntu-hpc%3Amatrix.org?logo=matrix&label=ubuntu-hpc)](https://matrix.to/#/#ubuntu-hpc:matrix.org)
+
+</div>
+
+The `slurm-charms` repository provides many of the SLURM daemons that are required to deploy, configure and manage a SLURM-based HPC cluster:
+
+- [`slurmctld`](./charms/slurmctld/): the central management daemon for SLURM.
+- [`slurmd`](./charms/slurmd): the compute node daemon for SLURM.
+- [`slurmdbd`](./charms/slurmdbd): the database daemon for SLURM.
+- [`slurmrestd`](./charms/slurmrestd/): the REST API interface to SLURM.
+
+## Usage
+
+This should be run using Juju 3.x or greater.
+
+#### Deploy a minimal Charmed SLURM cluster
+
+```shell
+$ juju deploy slurmctld --channel edge
+$ juju deploy slurmd --channel edge
+$ juju deploy slurmdbd --channel edge
+$ juju deploy slurmrestd --channel edge
+$ juju deploy mysql --channel 8.0/edge
```

Comment:
We should switch this to the stable branch. See https://github.com/charmed-hpc/slurm-bundles/issues/40:

```suggestion
$ juju deploy mysql --channel 8.0/stable
```